### PR TITLE
Nightingale sequence heatmap update to v1

### PR DESCRIPTION
**Pull request by marcelo-q**
Hello. The PDBe heatmap-component has been recently updated to it’s first stable version (1.0) and I’ve been asked to update the Nightingale Sequence Heatmap component to reflect this.

These are mostly minor fixes that do not affect what was previously discussed and adjusted in the first PR.

Perhaps the biggest change is the introduction of an attributed named "hm-highlight-width" to control the border width of the heatmap highlight.

Please let me know if there is anything requiring adjustment and I will be glad to reply and fix it

### CODE REVIEW

File: dev/index.html [hunk 1]
```diff
@@ -185,6 +190,7 @@ <h1>Manager</h1>
           height="500"
           display-start="5"
           display-end="25"
+          hm-highlight-width="0"
```
Comment (gustavo-salazar) [inline dev/index.html:193]: If I understand correctly, the only difference in not adding this attribute is the border of the highlighted area defaults to 1, isn't it?

File: dev/index.html [hunk 2]
```diff
@@ -124,6 +124,11 @@
           document
             .getElementById("heatmapHotmap")
             .setHeatmapData(xDomain, yDomain, data);
+
+          // await document.getElementById("heatmapHotmap").updateComplete.then(() => {
```
Comment (gustavo-salazar) [inline dev/index.html:128]: Delete this line if the are not going to be in use

**Review by gustavo-salazar — APPROVED** [review]
Hey there, 
Thanks for updating it to the latest version.
I put a couple of minor comments, but otherwise I'm happy to merge this. 
The question about the new attribute is mostly about deciding if this could be release as a minor version. i.e. not breaking changes

File: dev/index.html (hunk 2 above)
Comment (marcelo-q) [reply dev/index.html]: Done

File: dev/index.html (hunk 1 above)
Comment (marcelo-q) [reply dev/index.html:189]: Yes. This was added so the Nightingale Sequence Heatmap matches the highlighted style of Nightingale better after the PDBe heatmap-component fixes

**Comment by gustavo-salazar** [comment]
Hey thanks for the changes. before I merge, do you mind solving the conflict with `main`, I think it's because I ran prettier on the `main` branch after you have branch out

**Pull request merged**